libs/community/langchain_community/tools/dataherald/tool.py

```diff
@@ -0,0 +1,37 @@
+"""Tool for the Dataherald Hosted API"""
+
+from typing import Optional, Type
+
+from langchain_core.callbacks import CallbackManagerForToolRun
+from langchain_core.pydantic_v1 import BaseModel, Field
+from langchain_core.tools import BaseTool
+
+from langchain_community.utilities.dataherald import DataheraldAPIWrapper
+
+
+class DataheraldTextToSQLInput(BaseModel):
```

Comment:
add an input schema with a description for the prompt field, so that if you use a tool-calling llm this description gets added as part of the tool schema. feel free to change the wording!

Reply:
@baskaryan hi, I have a question about how to send another parameter, for example I need always the `prompt` and a `db_connection_id` (which was generated previously in our platform).

1. One option is adding it in the wrapper like this:
```
api_wrapper = DataheraldAPIWrapper(dataherald_api_key="", db_connection_id="")
```
besides `db_connection_id` could be a env var (DATAHERALD_DB_CONNECTION_ID) but if I want to change the db connection in the execution process could be difficult.

2. The other option is send the db connection when I invoke but not sure how to do it
```
agent_executor.invoke({"input":{"prompt":"foo", "db_connection_id": "bar"})
```
It doesn't take the `db_connection_id` param and raise an error.

Reply:
Hi, I followed this approach so the db_connection_id should be set when the Wrapper is created
`
api_wrapper = DataheraldAPIWrapper(dataherald_api_key="", db_connection_id="")
`

Reply:
yea think setting at initialization makes sense 👍

Reply:
Thanks, feel free to merge it if it pass all the validations